### PR TITLE
New version: Barry-ran.QtScrcpy version 2.2.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140898)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/b/Barry-ran/QtScrcpy/2.2.0/Barry-ran.QtScrcpy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+PackageIdentifier: Barry-ran.QtScrcpy
+PackageVersion: 2.2.0
+InstallerType: zip
+NestedInstallerType: portable
+ReleaseDate: 2024-02-25
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: QtScrcpy-win-x86-v2.1.2\QtScrcpy.exe
```
Comment (Exorcism0666) [inline manifests/b/Barry-ran/QtScrcpy/2.2.0/Barry-ran.QtScrcpy.installer.yaml:12]: @russellbanks "RelativeFilePath" is not updated

File: manifests/b/Barry-ran/QtScrcpy/2.2.0/Barry-ran.QtScrcpy.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+PublisherSupportUrl: https://github.com/barry-ran/QtScrcpy
+PrivacyUrl: https://github.com/barry-ran/QtScrcpy
+Author: barry-ran
+PackageName: QtScrcpy
+PackageUrl: https://github.com/barry-ran/QtScrcpy
+License: Apache-2.0 license
+LicenseUrl: https://github.com/barry-ran/QtScrcpy/blob/dev/LICENSE
+CopyrightUrl: https://github.com/barry-ran/QtScrcpy/blob/dev/LICENSE
+ShortDescription: 安卓实时显示控制软件
+ReleaseNotes: https://github.com/barry-ran/QtScrcpy/releases/tag/v2.1.2
```
Comment (Exorcism0666) [inline manifests/b/Barry-ran/QtScrcpy/2.2.0/Barry-ran.QtScrcpy.locale.zh-CN.yaml:18]: @russellbanks "ReleaseNotes" and "ReleaseNotesUrl" is not updated

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\QtScrcpy.exe returned exit code: -1073741515 Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\QtScrcpy.exe returned exit code: -1073741515

(Automated response - build 827.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**